linkup-cli/src/services/caddy.rs

```diff
@@ -185,15 +201,17 @@ impl Caddy {
         Ok(())
     }
 
-    fn check_redis_installed(&self) -> bool {
-        let output = Command::new("caddy").arg("list-modules").output().unwrap();
-
-        let output_str = String::from_utf8(output.stdout).unwrap();
+    pub fn should_start(&self, domains: &[String]) -> Result<bool, Error> {
+        for env_var in ["LINKUP_WORKER_TOKEN"] {
```

Comment:
As a next step, read this from config instead of env right?

Reply:
Oh! That is right! I forgot about that change. I think we can do that already in this PR. Will look into it.

Reply:
Actually, will take on a follow-up PR. It is a bit more than I thought.
But coming soon!